### ISSUE TITLE
add platform Citrix Xenserver as available platform

### ISSUE THREAD

**Issue by chouseknecht** [ansible/galaxy#84]
_From @johnpion on July 2, 2016 14:30_

I've written role for updating Citrix XenServer:  
https://galaxy.ansible.com/john-seregin/xenserver/

But I platforms like Xen, Xenserver, Citrix Xenserver don't available :(

Please add there

_Copied from original issue: ansible/galaxy-issues#166_

**Comment by vmpr** [comment]
yes please add XenServer as a platform and also the new XCP-ng fork
cheers, Ringo
